### PR TITLE
[Ubuntu] Adding latest Node.js version 22* .

### PR DESCRIPTION
**Pull request by hemanthmanga**
# Description
This PR adding latest Node.js version 22 in Ubuntu

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/ubuntu/toolsets/toolset-2404.json [hunk 1]
```diff
@@ -29,7 +29,8 @@
             "arch": "x64",
             "versions": [
                 "18.*",
-                "20.*"
+                "20.*",
+                "22.*"
```
Comment (erik-bershel) [inline images/ubuntu/toolsets/toolset-2404.json:33]: 1. Lets add new Node to all the supported platforms.
2. Check on the docker images section too for ubuntu-20 and ubuntu-22

Comment (hemanthmanga) [reply images/ubuntu/toolsets/toolset-2404.json:33]: Sure, I will do.

File: images/ubuntu/toolsets/toolset-2004.json [hunk 2]
```diff
@@ -223,6 +224,7 @@
             "node:16",
             "node:18",
             "node:20",
+            "node:22",
             "node:16-alpine",
             "node:18-alpine",
             "node:20-alpine",
```
Comment (erik-bershel) [inline images/ubuntu/toolsets/toolset-2004.json:230]:
```suggestion
            "node:20-alpine",
            "node:22-alpine"
```

File: images/ubuntu/toolsets/toolset-2204.json [hunk 3]
```diff
@@ -221,6 +222,7 @@
             "node:16",
             "node:18",
             "node:20",
+            "node:22",
             "node:16-alpine",
             "node:18-alpine",
             "node:20-alpine",
```
Comment (erik-bershel) [inline images/ubuntu/toolsets/toolset-2204.json:228]:
```suggestion
            "node:20-alpine",
            "node:22-alpine"
```

**Review by erik-bershel — DISMISSED** [review]

File: images/ubuntu/toolsets/toolset-2004.json [hunk 4]
```diff
@@ -223,9 +224,12 @@
...
+            "node:22-alpine"
+            "node:22-alpine",
```
Comment (erik-bershel) [inline images/ubuntu/toolsets/toolset-2004.json:232]:
```suggestion
            "node:22-alpine",
```
```suggestion
            "node:20-alpine",
            "node:22-alpine",
```

File: images/ubuntu/toolsets/toolset-2204.json [hunk 5]
```diff
@@ -221,9 +222,12 @@
...
+            "node:22-alpine"
+            "node:22-alpine",
```
Comment (erik-bershel) [inline images/ubuntu/toolsets/toolset-2204.json:230]:
```suggestion
            "node:20-alpine",
            "node:22-alpine",
```

**Review by erik-bershel — APPROVED** [review]

**Pull request merged**